task/task.json

```diff
@@ -40,6 +46,48 @@
       "required": true,
       "helpMarkDown": "Select the service endpoint for your GitHub connection, must be authorized with PAT. <br>To create one, click the Manage link and create a new service endpoint.",
       "visibleRule": "repoType=GitHub"
+    },
+	{
+      "name": "repositorySelector",
+      "type": "radio",
+      "label": "Repository to use",
+      "required": true,
+      "defaultValue": "currentBuild",
+      "options": {
+        "currentBuild": "Current repository build",
+        "select": "Select"
+      },
+      "helpMarkDown": "The method for selecting the Git repository. `Current build` will use the repository for which the current build is configured. `Select` will allow you to select an Azure Repository from your account.",
+      "groupName": "repository",
+      "visibleRule": "repoType=Azure DevOps"
```

Comment:
Try to remove the `visibleRule`, maybe it will solve the extension validation error.

Reply:
Hi,
Still the same error.
Try to remove the `visibleRule` from the `repositorySelector`.